Dockerfile

```
@@ -0,0 +1,3 @@
+FROM ruby:2.6-bullseye
+COPY Gemfile Gemfile.lock act_as_api_client.gemspec .
```

Hadolint

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)